optuna/integration/chainermn.py

```diff
@@ -1,27 +1,22 @@
+from datetime import datetime
+from typing import Any
+from typing import Callable
+from typing import Dict
 from typing import Optional
+from typing import Sequence
+from typing import Tuple
+from typing import Type
 import warnings
 
 from optuna._imports import try_import
 from optuna.storages import InMemoryStorage
 from optuna.storages import RDBStorage
 from optuna.trial import BaseTrial
 from optuna import TrialPruned
-from optuna import type_checking
-
-if type_checking.TYPE_CHECKING:
-    from datetime import datetime  # NOQA
-    from typing import Any  # NOQA
-    from typing import Callable  # NOQA
-    from typing import Dict  # NOQA
-    from typing import Sequence  # NOQA
-    from typing import Tuple  # NOQA
-    from typing import Type  # NOQA
-    from typing import Union  # NOQA
-
-    from optuna.distributions import BaseDistribution  # NOQA
-    from optuna.distributions import CategoricalChoiceType  # NOQA
-    from optuna.study import Study  # NOQA
-    from optuna.trial import Trial  # NOQA
+from optuna.distributions import BaseDistribution  # NOQA
+from optuna.distributions import CategoricalChoiceType  # NOQA
+from optuna.study import Study  # NOQA
+from optuna.trial import Trial  # NOQA
```

Comment:
`NOQA` is no longer necessary because we use it in the code. And also, please reorder the import lines related to `optuna` in alphabetical order.
```suggestion
from optuna.distributions import BaseDistribution  # NOQA
from optuna.distributions import CategoricalChoiceType  # NOQA
from optuna.study import Study  # NOQA
from optuna.trial import Trial  # NOQA
```